tests/test_column_info.py

```diff
@@ -65,42 +66,47 @@ def test_dataframe_input_schema_with_json_cols():
     assert list(input_df.columns) == raw_data_columns
 
     column_info = [
-        DateTimeColumn(name="timestamp", dtype=datetime, input_name="time"),
-        RenameColumn(name="userId", dtype=str, input_name="properties.userPrincipalName"),
-        RenameColumn(name="appDisplayName", dtype=str, input_name="properties.appDisplayName"),
-        ColumnInfo(name="category", dtype=str),
-        RenameColumn(name="clientAppUsed", dtype=str, input_name="properties.clientAppUsed"),
-        RenameColumn(name="deviceDetailbrowser", dtype=str, input_name="properties.deviceDetail.browser"),
-        RenameColumn(name="deviceDetaildisplayName", dtype=str, input_name="properties.deviceDetail.displayName"),
+        DateTimeColumn(name="timestamp", dtype='datetime64[ns]', input_name="time"),
+        RenameColumn(name="userId", dtype='str', input_name="properties.userPrincipalName"),
+        RenameColumn(name="appDisplayName", dtype='str', input_name="properties.appDisplayName"),
+        ColumnInfo(name="category", dtype='str'),
+        RenameColumn(name="clientAppUsed", dtype='str', input_name="properties.clientAppUsed"),
+        RenameColumn(name="deviceDetailbrowser", dtype='str', input_name="properties.deviceDetail.browser"),
+        RenameColumn(name="deviceDetaildisplayName", dtype='str', input_name="properties.deviceDetail.displayName"),
         RenameColumn(name="deviceDetailoperatingSystem",
-                     dtype=str,
+                     dtype='str',
                      input_name="properties.deviceDetail.operatingSystem"),
         StringCatColumn(name="location",
-                        dtype=str,
+                        dtype='str',
                         input_columns=[
                             "properties.location.city",
                             "properties.location.countryOrRegion",
                         ],
                         sep=", "),
-        RenameColumn(name="statusfailureReason", dtype=str, input_name="properties.status.failureReason"),
+        RenameColumn(name="statusfailureReason", dtype='str', input_name="properties.status.failureReason"),
```

Comment:
I think these should use their original type object to test conversion to strings

Reply:
That's not really their dtype though, its something that gets translated into a pandas/cudf type. It also gets returned by the ColumnInfo 'get_pandas_dtype' call.